node/node.go

```diff
@@ -412,7 +412,6 @@ func (node *AlgorandFullNode) BroadcastSignedTxGroup(txgroup []transactions.Sign
 	err = node.transactionPool.Remember(txgroup)
 	if err != nil {
 		node.log.Infof("rejected by local pool: %v - transaction group was %+v", err, txgroup)
```

Comment:
We'll probably want to add to the log message here that we're attempting to broadcast anyway, even though we've rejected the transaction from the pool.

Reply:
Updated info message to warning and added text to indicate that the transaction will be broadcast even though it was rejected from the pool.